cmake/templates/launch.json.in

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "cortex-debug",
+            "request": "launch",
+            "servertype": "stutil",
+            "cwd": "${CMAKE_CURRENT_SOURCE_DIR}",
+            "gdbPath": "arm-none-eabi-gdb",
+            "executable": "build/${PROJECT_NAME}.elf",
+            "name": "Cortex Debug (ST-Util)",
+            "device": "${DEVICE}",
+            "v1": false
+        },
+        {
+            "type": "cortex-debug",
+            "request": "launch",
+            "servertype": "jlink",
+            "cwd": "${CMAKE_CURRENT_SOURCE_DIR}",
+            "gdbPath": "arm-none-eabi-gdb",
+            "executable": "build/${PROJECT_NAME}.elf",
+            "name": "Cortex Debug (J-Link)",
+            "device": "${DEVICE}",
+            "interface": "swd",
```

Comment:
Acho que o padrão do gdb agora é o gdb-multiarch ao invés do arm-none-eabi-gdb, mas não sei se já da p mudar aqui no template pq acho q n atualizou p Ubuntu 20.04, mas é algo a se lembrar p qm for tentar debuggar dps.